pandas/tests/tseries/offsets/test_year.py

```diff
@@ -19,6 +20,29 @@
 )
 
 
+@pytest.mark.parametrize("n", [-2, 1])
+@pytest.mark.parametrize(
+    "cls",
+    [
+        YearBegin,
+        YearEnd,
+    ],
+)
+def test_apply_index(cls, n):
```

Comment:
we don't want to have this test duplicated in a bunch of places

Reply:
Totally understandable. Consolidated those into a different file.